docs/en/observability/whats-new.asciidoc

```diff
@@ -101,9 +101,9 @@ image::images/whats-new-slo-create-rule.png[]
 
 
 [discrete]
-== Enhanced rule definition with context attributes
+== Enhanced APM rule definition
 
-You can now enhance Observability alert rule definitions with context attributes. For example, alerts can be based on specific variables like transaction types, error groups, host names, or cloud region. This allows for more granular controls on alerting to reduce alert fatigue and make notifications more actionable.
+You can now enhance Observability APM rule definitions with additional dimensions to filter and group alerts. For example, alerts can be based on specific variables like transaction types, transaction names, and error groups. This allows for more granular controls on alerting to reduce alert fatigue and make notifications more actionable. You can also group alerts on specific fields which will breakdown the alerts based on the selected fields. For example, you can create a rule grouped by transaction name. This creates an alert for each transaction name in the service.
```

Comment:
I'm not following this description very well and wondering if the screenshot needs to be updated to show this?

Reply:
@dedemorton Thanks for catching that. Indeed screenshot needs to be updated. Here are the updated screenshots for each APM rule that was updated.

@mdbirnstiehl Please feel free to modify the description as needed, and let me know if you have any questions.

**Latency threshold rule**
<img width="487" alt="Screenshot 2023-07-14 at 12 24 40" src="https://github.com/elastic/observability-docs/assets/69037875/47106006-43b1-4256-a5de-6419ee4c366a">

**Failed transaction threshold rule**
<img width="490" alt="Screenshot 2023-07-14 at 12 23 05" src="https://github.com/elastic/observability-docs/assets/69037875/128bd28c-aadf-427c-946d-1504df177c14">

**Error count threshold rule**
<img width="487" alt="Screenshot 2023-07-14 at 12 20 56" src="https://github.com/elastic/observability-docs/assets/69037875/e397e0a2-02f9-4700-8fdb-ec4276e8ef1a">